### DISCUSSION TITLE
Permission Denied while running sample call.js

### DISCUSSION THREAD

**Discussion by txrubel** [fonoster/yaps#237] [Q&A] [answered]
Ref. https://github.com/fonoster/fonoster/issues/236

Thanks @psanders for your quick response and sorry for my mistake, its not a BUG or Feature Request so it should go here.

Actually I have logged in with fonoster auth:login  and added my project as default project. this command create a file(/root/.fonoster/config) as following,
![image](https://user-images.githubusercontent.com/38957915/142130869-4da2e618-59bd-48fa-a509-cb34509838c1.png)

Do I need to configure accessKeyId and accessKeySecret in any other places?

Thanks

**Comment by psanders** [comment, marked as answer]
You are right, the example was missing the credentials. I updated the [Make an outbound call](https://learn.fonoster.com/docs/getting_started/make_an_outbound_call) to show a complete example.

**Reply by txrubel** [reply]
Thanks @psanders for update the doc

**Reply by psanders** [reply]
For sure. Let me know of any other issues you find.

**Reply by rajeevkri** [reply]
I am using credentials part. Still getting permission denied error.
Can anyone help?

**Reply by psanders** [reply]
Shows the code with the issue.

**Reply by BrayanMnz** [reply]
As @rajeevkri, I'm still getting permission denied. 
Attached the output and the code being used.

![image](https://user-images.githubusercontent.com/61812255/143081412-5a303f5f-5042-46ef-a401-f5c775d13623.png)

![image](https://user-images.githubusercontent.com/61812255/143081604-6b2a216c-4f51-41ea-b9e8-e75fed6c435c.png)

**Reply by psanders** [reply]
The code shows that you are using the User's credentials. Instead, you need your Project's `accessKeyId` and `accessKeySecret`.

**Reply by psanders** [reply]
You can get the credentials from `~/.fonoster/config` or you can use the command `fonoster projects:renew [REF]` to get a new token.

**Reply by BrayanMnz** [reply]
As @psanders  said in previous reply, updating `accessKeyId` and `accessKeySecret` solve it.

**Comment by rajeevkri** [comment]
I am using credentials part. Still getting permission denied error.
Can anyone help?
